modules/assimp/register_types.cpp

```diff
@@ -44,7 +44,13 @@ static void _editor_init() {
 void register_assimp_types() {
 
 #ifdef TOOLS_ENABLED
+	ClassDB::APIType prev_api = ClassDB::get_current_api();
+	ClassDB::set_current_api(ClassDB::API_EDITOR);
```

Comment:
Wouldn't it be less error prone to have a `ClassDB::register_editor_class` that does this automatically?

Reply:
Perhaps, but this is the only place where it was needed so far.